### ISSUE TITLE
open/ssh not starting

### ISSUE THREAD

**Issue by jungle-boogie** [trueos/trueos-core#175]
Hi All,

`12.0-CURRENT FreeBSD 12.0-CURRENT #40 a8b0e29(drm-next-4.7): Tue Nov 22 16:43:26 UTC 2016 root@gauntlet:/usr/obj/usr/src/sys`

sshd doesn't seem to start, even though it's enabled in `/etc/rc.conf`, and I tried starting it in the control panel services tool. I installed `openssh-portable` and while I see the service in the same control panel, I cannot start it or have it start at boot.

without this started, I cannot ssh into the machine.

**Comment by kmoore134** [comment]
I need to port over an ssh init.d script for openssh-portable and the base system ssh. Stay tuned, this is why this repo isn't pushed mainline yet ;)

**Comment by jungle-boogie** [comment]
hi @kmoore134, not a problem! Thanks for the update on this.

**Comment by kmoore134** [comment]
Ok, update on this. We have 'ssh' scripts for the base system SSH already included. Try the following:

# service sshd start

If that works, you can enable it at boot with:

# rc-update add sshd default

I still need to add it for openssh-portable, so keeping this open for now.

**Comment by jungle-boogie** [comment]
hi @kmoore134,

That seems to do the trick to start it up and enabling at boot.

Thanks!

**Comment by jungle-boogie** [comment]
running `FreeBSD 12.0-CURRENT #3 cc46c55(drm-next-4.7): Wed Dec 14 21:25:14 UTC 2016`

```
% sudo openrc
Password:
 * Caching service dependencies ...      [ ok ]
openssh             | * Starting openssh ...     [ ok ]
```
service manager show openssh as `false` for running.

Why?

**Comment by jungle-boogie** [comment]
Hi,

 sshd                                                                                                                                                                                            [  stopped  ]

 openssh                                                                                                                                                                                         [ starting ]

after starting, it shows crashed.

**Comment by jungle-boogie** [comment]
> Ok, most likely it is starting and crashing due to sshd running on the same port?

Yes, this is accurate as I see openrc complaining about this during the boot process. But, why?
I want opensshd to be the ssh daemon.

**Comment by pkgdemon** [comment]
It sounds like you need to remove sshd from the boot runlevel so openssh can take over if I am not mistaken?  If so rc-update delete sshd boot should do it.

**Comment by pkgdemon** [comment]
Or rc-update delete sshd default if it is in default.
